### ISSUE TITLE
Export knex.raw('CURRENT_TIMESTAMP') for quick usage

### ISSUE THREAD

**Issue by ktmud** [tgriesser/knex#158]
Maybe knex should allow people to write like this:

```javascript
    table.dateTime('updated_at')
      .defaultTo(knex.CURRENT_TIMESTAMP)

    table.date('day')
      .defaultTo(knex.CURRENT_DATE)

   table.dateTime('timestamp')
      .defaultTo(Date.now)
```

**Comment by mjbrisebois** [comment]
`knex.raw` is not a portable or flexible solution.  The proposal above is exactly what I would call a portable and flexible solution.  Why is this issue closed?

**Comment by rhys-vdw** [comment]
@mjbrisebois Reopened. I'll accept a PR that adds constants to `knex` as suggested by @ktmud.

**Comment by squarejaw** [comment]
This seems to have been implemented by #372.
```js
table.dateTime('updated_at').defaultTo(knex.fn.now());
```
Might be nice to have some better documentation for this, though.

**Comment by squarejaw** [comment]
There's a PR out for documentation changes related to this: #1029

**Comment by rhys-vdw** [comment]
Closed by #1029.

**Comment by bennyn** [comment]
For people ending up here from Google. This is the way how to do it in 2017:

```javascript
const knexfile = require('../../knexfile');
const knex = require('knex')(knexfile.development);
const date = knex.fn.now();
```
